ledger-api/testing-utils/src/main/scala/com/digitalasset/ledger/api/testing/utils/AkkaBeforeAndAfterAll.scala

```diff
@@ -23,7 +23,7 @@ trait AkkaBeforeAndAfterAll extends BeforeAndAfterAll {
 
   private implicit lazy val executionContext: ExecutionContext =
     ExecutionContext.fromExecutorService(
-      Executors.newSingleThreadExecutor(
+      Executors.newCachedThreadPool(
```

Comment:
I'm glad this worked. In a subsequent PR, can you try removing it entirely and let the actor system use the global EC, to see if that also works?

Reply:
What are we trying to achieve with this change?

Reply:
@cocreature for one of our tests in the oem integration kit repo, one thread is not sufficient as it gets blocked and another one has to unblock it.